batch.R

```diff
@@ -171,11 +171,20 @@ for(i in 1:nrow(fileDF)) {
   write.csv(x = metrics, file = file.path(outputFolder, constitName, paste(constitSite, "modelMetrics.csv", sep = "_")), row.names = FALSE)
   
   #make predictions
-  annualPreds <- bind_rows(
+  annualSummary <- bind_rows(
+    summarizePreds(pred_rload, siteMeta, "annual", model.name = "rloadest"),
+    summarizePreds(pred_interp, siteMeta, "annual", model.name = "interpolation"),
+    summarizePreds(pred_comp, siteMeta, "annual", model.name = "composite"))
+  annualSummary <- reshape(annualSummary, idvar = "Water_Year", direction = "wide", 
+                           v.names = c("Flux_Rate","SE", "CI_lower", "CI_upper"), timevar = "model")
+  write.csv(x = annualSummary, file = file.path(outputFolder, constitName, "annual", paste0(constitSite, '.csv')), row.names=FALSE)
+  
+  multiYearSummary <- bind_rows(
     summarizePreds(pred_rload, siteMeta, "total", model.name = "rloadest"),
     summarizePreds(pred_interp, siteMeta, "total", model.name = "interpolation"),
     summarizePreds(pred_comp, siteMeta, "total", model.name = "composite"))
-  write.csv(x = annualPreds, file = file.path(outputFolder, constitName, "annual", paste0(constitSite, '.csv')), row.names=FALSE)
+  multiYearSummary <- reshape(multiYearSummary, idvar = "site.id", direction = "wide", v.names = "multi_year_avg", timevar = "model")
```

Comment:
Going old school with `stats::reshape` instead of `tidyr::spread`! I don't know any reason not to. Curious, though: why did you choose the `stats` version?